source/socks5-support/socks5.rst

```diff
@@ -0,0 +1,186 @@
+==============
+SOCKS5 Support
```

Comment:
It might be helpful to have a top-level section with an architecture diagram / explanation of what a SOCKS5 setup looks like. For example, it was not clear to me that establishing a connection to a server requires drivers to specify both the proxy host/port _and_ the destination host/port. All of this is probably explained in depth in the RFC, but it's not the most digestable source of information for engineers that need a quick high-level overview.

Reply:
I’ve added a short note to the Abstract section here, feel free to take a look. I don’t think we need to go into a ton of detail here about how proxies work, though.